src/util/getDefaultLayerTypes.js

```diff
@@ -40,5 +40,4 @@ export const getDefaultLayerTypes = () => [
         img: 'images/orgunits.png',
         opacity: 1,
     },
-    ...earthEngineLayers.filter((l) => !l.legacy),
```

Comment:
This file has lint errors

Reply:
Fix: [6a92e37](https://github.com/dhis2/maps-app/pull/3198/commits/6a92e3742d33f45e2f8dcfb104d683051a70de60)